packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -1637,4 +1637,29 @@ audio = model.autoencoder.decode(codes)[0].cpu()
 torchaudio.save("sample.wav", audio, model.autoencoder.sampling_rate)
 `,
 ];
+
+export const smolvla = (model: ModelData): string[] => [
+	`# git clone https://github.com/huggingface/lerobot.git
+cd lerobot
+conda create -y -n lerobot python=3.10
+conda activate lerobot
+
+# Install ffmpeg (required for video processing)
+conda install ffmpeg=7.1.1 -c conda-forge
+
+# Install LeRobot with the SmolVLA extra dependencies
+pip install -e ".[${model.id}]"
+
+# Launch finetuning on your dataset
+cd lerobot && python lerobot/scripts/train.py \\
+  --policy.path=lerobot/smolvla_base \\
+  --dataset.repo_id=lerobot/svla_so101_pickplace \\
+  --batch_size=64 \\
+  --steps=20000 \\
+  --output_dir=outputs/train/my_smolvla \\
+  --job_name=my_smolvla_training \\
+  --policy.device=cuda \\
+  --wandb.enable=true`,
+];
+
```

Comment:
We usually demonstrate simple inference snippets. Does that make sense for smolvla, or not really?

Reply:
_(originally posted in DMs on slack)_

> However, unfortunately we cannot use smolvla_base for inference directly, the only use case for smolvla_base for now is fine-tuning.
Interesting! What I would suggest:
- if `modelId is smolvla_base` => then provide the finetune command
- if `modelId != smolvla_base` => then provide the "how to use" command + the finetune command

Since `export const smolvla = (model: ModelData): string[] => [` is a method and you have access to `ModelData` you can easily add some simple logic like this to the snippet generation

I see there are 380+ models on https://huggingface.co/models?other=lerobot and only 1 is not finetuned (if I understand correctly?) so it would be a shame not to show the inference command for all of them. What do you think?